minifi/minifi-integration-tests/src/test/resources/logback.xml

```diff
@@ -55,7 +55,7 @@
     
     <!-- valid logging levels: TRACE, DEBUG, INFO, WARN, ERROR -->
     
-    <logger name="org.apache.nifi" level="INFO"/>
+    <logger name="org.apache.nifi" level="DEBUG"/>
```

Comment:
I understand it is useful for investigation but is it actually needed to be changed permanently. My guess is that it makes the log really noisy.

Reply:
Thanks, reverted.